js.js

```diff
@@ -0,0 +1,18 @@
+const body = document.querySelector('body');
+const span = document.querySelector('span');
+const storage = window.sessionStorage;
+
+body.addEventListener('click', clicked);
+
+let counter = storage.getItem('counter') || 0;
+addCounter(span);
```

Comment:
Move it below function declarations. Even with the hoisting, it's quite hard to read.

Reply:
done